migrations/versions/201509041515_f9bb2a03ceb_add_registration_tables.py

```diff
@@ -88,6 +100,7 @@ def upgrade():
 
 
 def downgrade():
+    op.drop_column('registration_form_items', 'current_data_id', schema='event_registration')
```

Comment:
Sure, but here we create the foreign key without explicit name so dropping the column is the correct way to remove the constraint.